benchmarks/fuse.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+import subprocess
+
+TEST_REPO = "https://github.com/dandisets/000007"
+TEST_TAG = "0.220126.1903"
+
+
+class FuseBenchmarks:
+    timeout = 3600
+
+    def setup_cache(self):
+        work_dir = Path("000007")
+        subprocess.run(["datalad", "install", TEST_REPO], check=True)
+        subprocess.run(["git", "checkout", TEST_TAG], cwd=str(work_dir), check=True)
```

Comment:
FTR, with 0.16 , datalad's clone can pass options to `git clone` so tag could be passed, e.g.

```shell
$> python3 -c 'import datalad.api as dl; dl.clone("https://github.com/dandisets/000007", git_clone_opts={"branch": "0.220126.1903"})'
[INFO   ] Remote origin not usable by git-annex; setting annex-ignore                                     
[INFO   ] https://github.com/dandisets/000007/config download failed: Not Found                           
[INFO   ] access to 1 dataset sibling dandi-dandisets-dropbox not auto-enabled, enable with:
| 		datalad siblings -d "/tmp/000007" enable -s dandi-dandisets-dropbox 
install(ok): /tmp/000007 (dataset)

$> git -C 000007/ branch
* (no branch)
  git-annex

$> git -C 000007/ describe
0.220126.1903
```
but we don't have anything like that for `install` yet ... just backreferencing https://github.com/datalad/datalad/pull/5662 here